### PR TITLE
Added support for Nomacs

### PR DESCRIPTION
**Pull request by TrigonaMinima**
https://github.com/nomacs/nomacs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,6 +270,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [ncmpcpp](http://ncmpcpp.rybczak.net)
   - [newsbeuter](http://newsbeuter.org/)
   - [ngrok](https://ngrok.com/)
+  - [Nomacs](https://github.com/nomacs/nomacs)
```
Comment (lra) [inline README.md]: Can you link to the homepage instead? http://nomacs.org/

Comment (TrigonaMinima) [inline README.md]: Changed.

**Comment by lra** [comment]
conflicts

**Comment by TrigonaMinima** [comment]
It should not give conflicts now.

**Pull request merged**